autoRetryFailedCRR-S3C.js

```diff
@@ -161,7 +161,14 @@ function _requeueObject(bucket, key, versionId, counters, cb) {
                 next(err);
             });
         },
-    ], cb);
+    ], err => {
+        if (err) {
+            log.error('error in _requeueObject waterfall',
+                      { error: err.message });
+            ++counters.errors;
```

Comment:
Errors are already counted when they happen in the inner call, so we should not count them a second time here.